src/main/kotlin/xyz/bluspring/kilt/loader/KiltLoader.kt

```diff
@@ -582,6 +618,7 @@ class KiltLoader {
     }
 
     fun loadMods() {
+        runBlocking { scanModJob.join() }
```

Comment:
Does the scan only begin running in the `loadMods` section? This job should by all means *never* be executed this late in the process.
The mod scanning process should already be fully running in `KiltMixinPlugin`, so it can get picked up by `KiltEarlyRiser` later to run the access transformers.

Reply:
The scan mods is executing off thread. I'm trying to find the point to sync. The two are the result